File: UCF HSPT Documents/2014/Solutions/absolute.c

#include <stdio.h>

int main() {
    // Open the input file for reading
    FILE* ifp = fopen("absolute.in", "r");

    // Get the number of values to check
    int n;
    fscanf(ifp, "%d", &n);

    // Loop over those "n" values
    int i;
    for (i = 0; i < n; i++) {
        // Read this value
        int v;
        fscanf(ifp, "%d", &v);

        // Output it accordingly (making positive numbers negative)
        if (v < 0)
            printf("Integer #%d: %d\n", i+1, v);
        else
            printf("Integer #%d: %d\n", i+1, -v);
    }

    // Close the file and return
    fclose(ifp);
    return 0;
}

File: UCF HSPT Project/src/y2014/absolute.java

import java.io.File;
import java.io.IOException;
import java.util.Scanner;

public class absolute {
   public static void main(String[] args) throws IOException
   {
      // Open file for input
      Scanner input = new Scanner(new File("absolute.in"));

      // Read in number of test cases
      int T = input.nextInt();
      for (int t = 0; t < T; t++)
      {
         // Read in the number and then output its negative form
         int n = input.nextInt();
         System.out.printf("Integer #%d: %d%n", t+1, -Math.abs(n));
      }

      // Close file
      input.close();
   }
}
